google/cloud/aiplatform/explain/lit.py

```diff
@@ -15,11 +15,13 @@
 # limitations under the License.
 
 from typing import Dict, List, Tuple, Union
+import os
```

Comment:
Imports should be in lexicographic order.

Reply:
Done.